docs/quickstart/index.rst

```diff
@@ -138,9 +140,17 @@ Set a password for lemur user inside Postgres:
 .. code-block:: bash
 
     $ sudo -u postgres psql postgres
+    \CREATE USER lemur WITH PASSWORD 'lemur';
+    \GRANT ALL PRIVILEGES ON DATABASE "lemur" to lemur;
     \password lemur
     Enter new password: lemur
     Enter it again: lemur
+    \connect lemur
+    create table encrypted_keys (ID int primary key not null);
```

Comment:
I've known about this particular problem and I don't know if documentation is the ideal, way to fix it.  In essence, this migration https://github.com/Netflix/lemur/blob/master/lemur/migrations/versions/ed422fc58ba_.py#L35 introduces these temporary tables to migrate the key to a new format. You said that on 'clean' systems the lack of these tables cases the Lemur to fail to start? Ideally unless you are running ```lemur db migrate``` these migrations won't get run. I would be interested to see in which part of the process this fails for you. 

If you ran the ```lemur init``` command this command should have prevented the running of migrations:
https://github.com/Netflix/lemur/blob/master/lemur/manage.py#L128